docs/pages/application-access/okta/hosted-guide.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Setting up the Okta Service as a hosted integration
+description: How to set up the Okta Service as a hosted integration
+---
+
+## Prerequisites
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+
+- (!docs/pages/includes/tctl.mdx!)
+- An Okta organization with an [Okta API token](https://developer.okta.com/docs/guides/create-an-api-token) created.
+
+<Details title="Okta API token permissions">
+(!docs/pages/includes/okta-permissions.mdx hosted="true"!)
+</Details>
+
+### Enrolling a hosted Okta Integration
+
+
+#### The Enrolment Process
+
+Enrolling the Okta hosted integration will automatically perform the following
+actions:
+
+ 1. Create an Okta SAML Application called `Teleport $clustername` in the
+    upstream Okta organization
+ 1. Assign the Okta `Everyone` group to the new SAML Application
+ 1. Download the IdP Metadata for the new SAML Application
+ 1. Create a new SAML Authentication Connector called `okta-integration` in the
+    Teleport cluster
+ 1. Configure the Okta integration to draw its user pool from the Okta users
+    assigned to the new `Teleport $clustername` Okta SAML application
+ 1. Start the hosted Okta integration, including the Okta synchronization
+    process
```

Comment:
Wrong numbering: 
```suggestion
 1. Create an Okta SAML Application called `Teleport $clustername` in the
    upstream Okta organization
 2. Assign the Okta `Everyone` group to the new SAML Application
 3. Download the IdP Metadata for the new SAML Application
 4. Create a new SAML Authentication Connector called `okta-integration` in the
    Teleport cluster
 5. Configure the Okta integration to draw its user pool from the Okta users
    assigned to the new `Teleport $clustername` Okta SAML application
 6. Start the hosted Okta integration, including the Okta synchronization
    process
```

Reply:
I think this numbering scheme is ok, markdown will convert it to proper numbered list.

Reply:
This is actually correct—we require all ordered list items to start with `1.` so lists are easier to edit.

Reply:
I remember seeing something that our preferred style was all `1.` - but that was a whil ago and opinions may have changed. I personally prefer "real" numbers.